### PR TITLE
CB-10730 Move plugin handlers tests into platform's repo

### PR DESCRIPTION
**Pull request by vladimir-kotikov**
JIRA: https://issues.apache.org/jira/browse/CB-10730

### CODE REVIEW

File: spec/unit/JsprojManager.spec.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ var AppxManifest = require('../../template/cordova/lib/AppxManifest');
 var JsprojManager = rewire('../../template/cordova/lib/JsprojManager');
 
 var PROJECT_PATH = 'spec/unit/fixtures/EmptyProject';
```
Comment (dblotsky) [inline spec/unit/JsprojManager.spec.js]: Should this be `DummyProject`?

Comment (vladimir-kotikov) [inline spec/unit/JsprojManager.spec.js]: Updated

File: spec/unit/JsprojManager.spec.js [hunk 2]
```diff
@@ -53,6 +54,13 @@ describe('JsprojManager', function () {
         JsprojManager.__set__('proj', origProj);
     });
 
+    it('should throw if project is not an windows project', function () {
```
Comment (dblotsky) [inline spec/unit/JsprojManager.spec.js]: `"... not a windows ..."`

**Comment by dblotsky** [comment]
Is there also a PR for removing these from `cordova-lib`?

**Comment by vladimir-kotikov** [comment]
> Is there also a PR for removing these from cordova-lib?

No, because the code these tests are testing is still in cordova-lib. I think we could remove the tests along with the code as a part of PlatformApi polyfill deprecation (see [CB-10644](https://issues.apache.org/jira/browse/CB-10644) and corresponding [mailing list discussion](http://markmail.org/thread/b5j4mhvnxws7jbgk) )

**Pull request closed without merging**